falcon/routing/converters.py

```diff
@@ -59,6 +65,28 @@ def convert(self, fragment):
         return value
 
 
+class DateTimeConverter(object):
+    """Converts a field value to a datetime.
+
+    Keyword Args:
+        format_string (str): String used to parse the param value
+            into a datetime. Any format recognized by strptime() is
+            supported (default ``'%Y-%m-%dT%H:%M:%SZ'``).
+    """
+
+    __slots__ = ('_format_string', '_strptime')
```

Comment:
👍  Nice catch! This was something leftover from an experiment that I forgot to remove.